guidelines/MONAI-Workloads.md

```diff
@@ -0,0 +1,1149 @@
+3. DICOM router sends the original image to be stored in PACS and also in parallel routes a copy of the study to an AI processing engine.  DICOM router and AI processing engine are connected to each other with a high speed 10gbps network connection.  
+4. DICOM study gets processed by the AI processing engine, including pre-processing, inference and post processing.  The output of AI processing will be a probability score of the patient having COVID19.  The result will be packaged into a DICOM SR object.  The size of the resulting object will be the size of a DICOM image.  
+5. AI processing engine sends the study further to DICOM routing.
+6. DICOM router sends the study further to PACS 
+7. PACS or the AI processing engine notifies RIS about the AI diagnosis being ready in the radiology worklist. 
+8. The radiologist and/or physician responsible for ordering the study or reviewing the results sees the case come up on top of their worklist with both the original DICOM image and the AI provided scoring.  
+
+**Clinical Use Case:** prospective diagnostics
+
+**Computational Workload Type:** [Asynchronous](#asynchronous-computational-workload)
+
+**Clinical workload:** 1,500 studies per day across all modalities with less 18 minutes per year of time required for manual intervention and less than 1% failures in processing.  Study size N x M x K depending on modality and requested study type.  See Table for estimates of study size to modality and study type mapping.
+
+**Importance to Clinical AI Deployment:** high, the implementation of AI as part of scheduled radiology diagnosis will have a significant impact on patient care
+
+
+**Figure:** Scheduled Radiology Sequence
+
+![MONAI_Scheduled Radiology](assets/MONAI_Scheduled_Radiology.jpg)
+
+
+#### Interactive Diagnostics During Image Reading  {#interactive-diagnostics-during-image-reading}
+
+While a radiologist is reading a particular study, they request an AI algorithm to analyze it.
+
+A time delay of 10 secs might be acceptable in _edge cases_, where the radiologist themselves needs a second opinion from fellow radiologists. 
+
+**Classifying nodules (localised anomaly) in a 3D scan**
+
+Imagine a 3D image where a radiologist only wants to get an inference on a small section of an image. The small section may be free hand drawn by the radiologist a selected 3D cube (or a 2D rectangular region). An inference will be done only in the selected region. Annotating this region will generate a DICOM SR object and only the SR object will be sent to the inference engine. The inference will be computed and a SR object will be sent back.   
+
+**<span style="text-decoration:underline;">Inference on only a single series in a 4D image</span>**
```

Comment:
this is bad form, please don't do either of these things:
- use HTML in a markdown document because it's intended for consuming as plain-text as well as markup.
- underline text which isn't a hyperlink

Reply:
This is an artifact of the conversion tool.  I've cleaned up a few, but others were scattered throughout the document by other authors.